packages/marketplace/src/reducers/developer-detail.ts

```diff
@@ -0,0 +1,50 @@
+import { Action } from '../types/core'
+import { isType } from '../utils/actions'
+import {} from '../actions/developer-detail'
+import { DeveloperModel } from '@reapit/foundations-ts-definitions'
+import {
+  developerDetailLoading,
+  developerDetailReceiveData,
+  developerDetailRequestDataFailure,
+} from '../actions/developer-detail'
+
+export interface DeveloperDetailState {
+  loading: boolean
+  developerDetailData: DeveloperModel | null
+}
+
+export const defaultState: DeveloperDetailState = {
+  loading: false,
+  developerDetailData: null,
+}
+
+const developerDetailReducer = (
+  state: DeveloperDetailState = defaultState,
+  action: Action<any>,
+): DeveloperDetailState => {
+  if (isType(action, developerDetailLoading)) {
+    return {
+      ...state,
+      loading: action.data,
+    }
+  }
+
+  if (isType(action, developerDetailReceiveData)) {
+    return {
+      ...state,
+      loading: false,
+      developerDetailData: action.data || null,
```

Comment:
This one also has fetchMyIdentity Action, pls reuse that instead of writing the new things